application/helpers/userstatistics_helper.php

```diff
@@ -2882,16 +2882,17 @@ public function generate_statistics($surveyid, $allfields, $q2show = 'all', $use
 
                 break;
             case 'html':
-                $sGoogleMapsAPIKey = trim(Yii::app()->getConfig("googleMapsAPIKey"));
-                if ($sGoogleMapsAPIKey != '') {
-                    $sGoogleMapsAPIKey = '&key='.$sGoogleMapsAPIKey;
-                }
                 $sSSL = '';
                 if (!empty($_SERVER['HTTPS']) && $_SERVER['HTTPS'] != "off") {
                     $sSSL = 's';
                 }
-                $sOutputHTML .= "<script type=\"text/javascript\" src=\"http{$sSSL}://maps.googleapis.com/maps/api/js?sensor=false$sGoogleMapsAPIKey\"></script>\n"
-                ."<script type=\"text/javascript\">var site_url='".Yii::app()->baseUrl."';var temppath='".Yii::app()->getConfig("tempurl")."';var imgpath='".Yii::app()->getConfig('adminimageurl')."';var aStatData=".ls_json_encode($aStatisticsData)."</script>";
+                $sGoogleMapsAPIKey = trim(Yii::app()->getConfig("googleMapsAPIKey"));
+                if ($sGoogleMapsAPIKey != '') {
+                    $sGoogleMapsAPIKey = '&key='.$sGoogleMapsAPIKey;
+                    $sOutputHTML .= "<script type=\"text/javascript\" src=\"http{$sSSL}://maps.googleapis.com/maps/api/js?sensor=false$sGoogleMapsAPIKey\"></script>\n";
```

Comment:
`rc=\"//maps.googleapis.com/maps/api/js?sensor=false$sGoogleMapsAPIKey\">` must be OK, no need checking ssl or not (browser take current scheme).